### PR TITLE
feature/SC-4056/timetable check for events

### PR DESCRIPTION
**Pull request by pauludobeneke**
Screen Shot 2020-05-13 at 11 15 58

# Description

## Links to Tickets or other pull requests
Base links to copy
- https://ticketsystem.schul-cloud.org/browse/SC-4056

## Changes
- reactivate timetable

## Approval for review
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definiton of Done
More and detailed information on the *definition of done* can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

**Comment by malcm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5441468&bid=17844214)

**Comment by malcm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5441468&bid=17844214)

**Comment by CeEv** [comment]
Please fix codacy message.

**Comment by malcm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5441468&bid=17844214)

**Comment by malcm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5441468&bid=17844214)

File: views/dashboard/dashboard.hbs [hunk 1]
```diff
@@ -17,49 +17,50 @@
...
-                {{#times 11}}
-                <div class="timelines" style="width:{{this}}0%"></div>
-                {{/times}}
             </div>
         </div>
-    </div>
-	-->
+    {{else}}
+        <img class="w-100 pl-1 pr-1 svg-timetable-empty" src="/images/empty-states/timetable-empty-cards.svg" alt="{{$t "dashboard.text.emptyTimetable"}}"/>
+        <p class="my-2 pl-1 text-muted">{{$t "dashboard.text.empytTimetableInfo"}}</p>
```
Comment (dominikbruechner) [inline views/dashboard/dashboard.hbs:62]: we need to change the i18n key here

**Comment by malcm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5441468&bid=17844214)

**Comment by malcm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
- Added 42
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5441468&bid=17844214)

**Comment by malcm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
- Added 42
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5441468&bid=17844214)

**Comment by janrenz** [comment]
Bildschirmfoto 2020-05-30 um 10 37 59

File: controllers/dashboard.js [hunk 2]
```diff
@@ -36,28 +36,24 @@ router.get('/', (req, res, next) => {
...
-			// now we have to load all events from the beginning of time
-			// until end of the current day, map recurring events and
-			// display only the correct ones.
-			// I'm not happy with the solution but don't see any other less
-			// crappy way for this without changing the
-			// calendar service in it's core.
+	const eventsPromise = api(req)
+		.get('/calendar/', {
+			qs: {
+				all: 'true', // lets show all dates in dashboard
```
Comment (CeEv) [inline controllers/dashboard.js:47]: All true is set by default.

**Pull request merged**